src/smartpeak/source/core/RawDataProcessor.cpp

```diff
@@ -2646,6 +2674,9 @@ namespace SmartPeak
 
     rawDataHandler_IO.setFeatureMap(feat_map);
     rawDataHandler_IO.updateFeatureMapHistory();
+    for (auto& chromatogram : merged_chromatograms) {
```

Comment:
I would recommend replacing this with `rawDataHandler_IO.getExperiment().setChromatograms(chromatogram);`.   I may have mispoken earlier today so I apologize if that was the case.